docs/api/graphql/query.mdx

```diff
@@ -14,6 +14,10 @@ A request handler that captures a GraphQL query by the given name.
   for the endpoint-based mocking.
 </Hint>
 
+<Hint mode="warning">
+  If you’re using Apollo Client, you may have to add `__typename`s to mock responses.
```

Comment:
Could you please wrap "Apollo Client" in a link to their website/docs? Good for discovery.